.github/workflows/claude-org.yml

```
@@ -0,0 +1,68 @@
+  issues:
+    types: [opened, assigned]
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  claude:
+    env:
+      # Define authorized users (space-separated for easy contains() check)
+      # The surrounding spaces prevent partial matches (e.g., "admin" won't match "badmin")
+      AUTHORIZED_USERS: ' yekkhan-liftoff wyangsun tommynguyen-vungle '
+    if: |
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)